packages/react-components/src/components/video/index.tsx

```diff
@@ -191,6 +191,13 @@ const Video = ({
             if (!isNaN(volume)) {
                 el.volume = volume
             }
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [])
```

Comment:
@matheusmorett2 this is where `setVideoEl` is now initiated.

Reply:
👍